### PR TITLE
change to using ISO date

### PR DESCRIPTION
**Pull request by rudigiesler**

### CODE REVIEW

**Comment by hodgestar** [comment]
I think we need to update the tests too?

**Comment by rudigiesler** [comment]
There are no tests for this, because we replace the `now` function with one that returns a constant, to allow us to test. We could maybe do some type of regexp test? @hodgestar what do you think is the best way to test this?

**Comment by hodgestar** [comment]
Ready for review.

File: test/app.test.js [hunk 1]
```diff
@@ -74,6 +72,25 @@ describe("App", function() {
...
+                assert.equal(app.now(), "1970-01-01T00:00:01.234Z");
+            });
+        });
+
+        describe("when a no timestamp is set", function() {
+            it("should return the current time", function() {
+                delete app.now.timestamp;
+                assert.equal(
+                    (new RegExp('^\\d{4}-\\d{2}-\\d{2}T\\d{2}' +
+                                ':\\d{2}:\\d{2}.\\d{3}Z$'))
```
Comment (rudigiesler) [inline test/app.test.js]: Needs to be `\\.` to avoid matching any characters.

**Comment by rudigiesler** [comment]
:+1:

**Pull request closed without merging**